src/AppBundle/Entity/Report/Traits/ReportProfDeputyCostsEstimateTrait.php

```diff
@@ -32,6 +33,20 @@
      */
     private $profDeputyEstimateCostTypeIds;
 
+    /**
+     * @var float
+     *
+     * @JMS\Type("double")
+     * @JMS\Groups({"prof-deputy-management-costs"})
+     */
+    private $profDeputyManagementCosts;
```

Comment:
rename this to be the amount only as detailed above. No 's' at the end as its a 1:1 relationship